microservices/submission-service/src/replay-validation/replay-validation.module.ts

```diff
@@ -1,7 +1,4 @@
 import {forwardRef, Module} from "@nestjs/common";
-import {
```

Comment:
Don't delete the whole line! Just `UtilModule`

`import {
CoreModule, MatchmakingModule, MinioModule
} from "@sprocketbot/common";

Reply:
i blame DST